torch/linalg/__init__.py

```diff
@@ -833,19 +833,62 @@
 If :attr:`hermitian` is `True` then transpose operation is the conjugate transpose.
 
 :math:`L` (or :math:`U`) and :math:`D` are stored in compact form in ``LD``.
-They follow the format specified by LAPACK's `sytrf' function.
+They follow the format specified by `LAPACK's sytrf`_ function.
 These tensors may be used in :func:`torch.linalg.ldl_solve` to solve linear systems.
 
-``info`` stores integer error codes from the backend library (see `sytrf` function from LAPACK).
-The positive integer indicates the diagonal element of :math:`D` that is zero.
-Division by 0 will occur if the result is used for solving a system of linear equations.
-``info`` filled with zeros indicates that the factorization was successful.
-If ``check_errors=True`` and ``info`` contains positive integers, then a RuntimeError is thrown.
-
 Supports input of float, double, cfloat and cdouble dtypes.
 Also supports batches of matrices, and if :attr:`A` is a batch of matrices then
 the output has the same batch dimensions.
 
+""" + fr"""
+.. note:: {common_notes["sync_note_has_ex"].format("torch.linalg.ldl_factor_ex")}
+
+.. warning:: {common_notes["experimental_warning"]}
```

Comment:
This line should be in the `_ex` variant.

Reply:
It's a new function (`torch.linalg.ldl_factor`) that doesn't exist in NumPy, SciPy, and Array API. So I thought it would be fine to have this note:
> This function is "experimental" and it may change in a future PyTorch release.

Reply:
IMO, this function is very sound and it's here to stay, so I think it's fine to skip this note and just put it in the `_ex` variant.

Reply:
Okay, I'll remove it.